src/instance/minecraft/chat-manager.ts

```diff
@@ -101,8 +103,12 @@ export default class ChatManager extends EventHandler<MinecraftInstance> {
         assert(message) // old packet means message exist
         resultMessage = prismChat.fromNotch(message)
       }
-
-      this.onMessage(resultMessage.toString())
+      this.onMessage(
+        this.proceedFiltering(
+          (data as { senderName?: string }).senderName ?? 'unknown sender',
+          resultMessage.toString()
+        )
+      )
```

Comment:
This is RAW chat manager. You can't just try filtering the RAW chat. RAW chat are meant to be raw and to stay that way. Message filtering should be done after the raw message has been identified as a guild user message and parsed. This happens in `./chat/public.ts` https://github.com/aidn3/hypixel-guild-discord-bridge/blob/d1c82cef5273d600657a8b50fd301d2ddc4e5bb2/src/instance/minecraft/chat/public.ts#L37-L46